src/commands/Util/MoneyCommand.js

```diff
@@ -39,8 +39,9 @@ class LeaveCommand extends Command {
     try {
       const res = await ExchangeApi.getInstance().getExchange(from.toUpperCase(), to.toUpperCase());
       const total = (res.rates[to] * parseFloat(ammount)).toFixed(2);
+      const totalFormated = Intl.NumberFormat('pt-BR', { style: 'currency', currency: to}).format(total)
```

Comment:
O Intl utiliza o segundo parâmetro { style: 'currency', currency: to} para formatar o valor e ignora o primeiro o que permite que o valor fique fixo em pt-BR. Acham necessário utilizar o primeiro parâmetro da config do bot?